src/vellum/workflows/runner/runner.py

```diff
@@ -132,7 +133,14 @@ def _emit_event(self, event: WorkflowEvent) -> WorkflowEvent:
         self._background_thread_queue.put(event)
         return event
 
+    @wrapper_execution_parent_context(
+        lambda self: WorkflowParentContext(
+            workflow_definition=self.workflow.__class__,
+            span_id=self._initial_state.meta.span_id,
+        )
+    )
     def _run_work_item(self, node: BaseNode[StateType], span_id: UUID) -> None:
+        parent_context = get_execution_context().get('parent_context')
```

Comment:
discussion topic: doing something like `with get_execution_context() as parent_context:` here

Reply:
@dvargas92495 the downside of that is it adds additional layer of indentation
I'd be a big fan of being able to do `.parent_context` instead of `.get("parent_context")`